### ISSUE TITLE
Container version update introduces breaking changes

### ISSUE THREAD

**Issue by numericalatina** [bitnami/bitnami-docker-moodle#142]
Latest image build break existing installations. Could be because of a change in the needed volume definitions.
Could be related to #141 

**Steps to reproduce the issue:**

1. Running image version **3.9.1**(sha256:3924b33c5608c88566082435a2e058e6d7c110a4c7d839f61009a4e6e50ad5b9)
2. A simple **docker stack deploy**, causing the **update** of the underlying docker image (because 3.9.1 is a _rolling tag_)
3. Containers end up using version sha256:c6e5bdaa3ba25bc9e513233fccc9291cbd3c7720371967313db44e302fef9a21

**Describe the results you received:**

The containers cannot start after the update

According to the logs obtained entering the container, the issue could be caused because of a redefinition of where **moodle data** should be stored:
```
WARN  ==> Found 'moodledata' directory inside /bitnami/moodle. Support for this configuration is deprecated and will be removed soon. Please create a new volume mountpoint at /bitnami/moodledata, and copy all its files there.
INFO  ==> Running database upgrade
find: ‘/bitnami/moodledata/sessions/’: No such file or directory
```

**Describe the results you expected:**

Would have expected containers to be backwards-compatible.
Or at least to have found an explanation on the release notes, if breaking changes have been introduced.

**Version**

- Output of `docker version`:

```
Server: Docker Engine - Community
 Engine:
  Version:          19.03.4
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.10
  Git commit:       9013bf583a
  Built:            Fri Oct 18 15:51:05 2019
  OS/Arch:          linux/amd64

```

**Comment by marcosbc** [comment]
Hi @numericalatina, you seem to be right, we'll look into the specific issue since we originally implemented the change with the purpose of supporting previous installs.

In the meantime, note that all Bitnami Moodle Docker images with rolling tag 3.9.1 include this change, while Moodle images with rolling tag 3.9.0 won't, so in the meantime you could go back to 3.9.0.

Alternatively (and to migrate to the final approach), you could always copy (with `cp -rp`) all files inside /bitnami/moodle/moodledata to /bitnami/moodledata and it should start working from now on.

To do that, you could run a Blackbox container image in a similar way like we describe how to backup the container files: https://github.com/bitnami/bitnami-docker-moodle#step-2-run-the-backup-command

**Comment by marcosbc** [comment]
Hi @numericalatina, we believe to have fixed this issue, and you should find the fix in 3.9.2-debian-10-r25 later today or early tomorrow.

**Comment by marcosbc** [comment]
Hi, just wanted to let you know that Moodle 3.9.2-debian-10-r25 was released with the fix. Could you try again and let us know if it works?
